src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectionTestWithSSLCert/CertificateTest.cs

```diff
@@ -142,8 +141,8 @@ public void OpeningConnectionWitHNICTest()
             connection.Open();
             Assert.Equal(ConnectionState.Open, connection.State);
 
-            // Ipv6 however causes name mistmatch error
-            // In net6 Manged SNI does not check for SAN. Therefore Application using Net6 have to use FQDN as HNIC
+            // Ipv6 however causes name mismatch error
+            // In net6 Manged SNI does not check for SAN. Therefor Application using Net6 have to use FQDN as HNIC
             // According to above no other hostname in certificate than FQDN will work in net6 which is same as SubjectName in case of RemoteCertificateNameMismatch
             // Net7.0 the new API added by dotnet runtime will check SANS and then SubjectName
```

Comment:
With the removal of .NET 6.0 support, I'd question how relevant this comment is, and whether anything else in the test needs to change...